### DISCUSSION TITLE
How to tell if Multizone is working / not working?

### DISCUSSION THREAD

**Discussion by mbze430** [seerge/g-helper#915] [Q&A]
I have been toggling the multizone button and I can't tell there differences.  with the brightness all the way down.  over a black background I can see the halo around the mouse cursor regardless multizone button is on or off.  I have my screen on HDR.

**Comment by seerge** [comment]
If you have HDR turned ON in windows, display will be always in Multizone (no matter of settings). I think AC even disables multizone button in that case.

If HDR is off - i assume you should see a difference

**Reply by IceStormNG** [reply]
AC actually leaves it there (at least on the version I used a few months ago), but if you toggle it, it disables HDR and switches back to SDR.

**Comment by mbze430** [comment]
uhmm I didn't realize that.  I removed AC the second I got the laptop.  Since everyone on the reddit said it was junk.  But at least I now know single zone is only available in SDR.

Thank!
